packages/documentation/copy/en/project-config/tsconfig.json.md

```diff
@@ -95,8 +95,9 @@ For example, if you were writing a project which uses Node.js version 12 and abo
 This lets your `tsconfig.json` focus on the unique choices for your project, and not all of the runtime mechanics. There are a few tsconfig bases already, and we're hoping the community can add more for different environments.
 
 - [Recommended](https://www.npmjs.com/package/@tsconfig/recommended)
```

Comment:
Is there any reason to keep this list at all when the intro statement just links to the repo?